src/foam/nanos/auth/CachingAuthService.java

```diff
@@ -66,7 +68,16 @@ public void eof() {
         public void reset(Detachable sub) {
           purgeCache(x);
         }
-      }, TRUE);
+      };
+
+      DAO userDAO       = (DAO) x.get("userDAO");
```

Comment:
We should be listening on `localGroupDAO` and `localGroupDAO` so anything that accesses those directly will still cause the listeners to trigger.

Reply:
Switch to local versions. Is that what you're saying?

Reply:
Yeah